### PR TITLE
Minio S3 setup for tests.

### PR DESCRIPTION
**Pull request by rasapala**
Minio S3 setup for tests.

### CODE REVIEW

File: tests/functional/utils/model_management.py [hunk 1]
```diff
@@ -40,6 +40,23 @@ def wait_endpoint_setup(container):
     time.sleep(2)
     return running
 
+def wait_minio_endpoint_setup(container):
```
Comment (joannanosek) [inline tests/functional/utils/model_management.py]: This function and `wait_endpoint_setup` from this file could be merged to one with a parameter as a message we want to have in logs.

Comment (rasapala) [reply tests/functional/utils/model_management.py]: wait_endpoint_setup checks for specific line in logs and has a big 900 timeout while wait_minio_endpoint_setup checks container status and has smaller timeout thus I would prefer to leave it as separate functions.

Comment (joannanosek) [reply tests/functional/utils/model_management.py]: So the timeout can be parametrized too.
As long as `if` is checking for boolean value, still the conditions as well as timeout can be passed as parameters, so the signature will looks like`def wait_endpoint_setup(condition, timeout)`.

Comment (rasapala) [reply tests/functional/utils/model_management.py]: Done.

File: tests/functional/utils/model_management.py [hunk 2]
```diff
@@ -21,21 +21,32 @@
 from pathlib import Path
 
 
-def wait_endpoint_setup(container):
+def minio_condition(container):
+    if "created" in container.status:
```
Comment (joannanosek) [inline tests/functional/utils/model_management.py]: `return "created" in container.status` will be enough, the same for `serving_condition`

**Pull request merged**